_includes/js/slider.js

```diff
@@ -99,8 +99,21 @@ for (let slider of sliders) {
 
   observer.observe(slider);
 
-  let firstSliderItem = slider.querySelector('.slider-item');
-  let firstSliderItemContent = slider.querySelector('.slider-item-content');
+  var firstSliderItem = slider.querySelector('.slider-item');
+  var firstSliderItemContent = slider.querySelector('.slider-item-content');
+
+  // If the slider is the agenda slider, show today's tab
+  if (slider.id === 'agenda-slider') {
```

Comment:
I guess to keep it more general, this should be based on some other condition, e.g., presence of some class.

Reply:
But for now, it's ok